settings/blt.settings.php

```diff
@@ -148,9 +148,15 @@
     // The hostname must match the pattern local.[sitename].com, where
     // [sitename] is a value in the multisites array.
     $domain_fragments = explode('.', $http_host);
-    $name = array_slice($domain_fragments, 1);
+    $name = $domain_fragments[1];
     $acsf_sites = $blt_config->get('multisites');
     if (in_array($name, $acsf_sites)) {
+
+      // Create a valid machine name based on the local domain name.
+      $name = strtolower($name);
+      $name = preg_replace('/[^a-z0-9_]+/', '_', $name);
+      $name = preg_replace('/_+/', '_', $name);
+
       $_acsf_site_name = $name;
```

Comment:
This is used in other places so I'm hesitant to mess with it. Might be better to do this when creating the split name in config.settings.php.